### PR TITLE
Fix lint setting

### PR DESCRIPTION
**Pull request by kitagry**
It didn't seem to work flake8, yapf and isort.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -33,9 +33,9 @@ lupa = "*"
 fakeredis = "*"
 
 [tool.flake8]
-# B006: Do not use mutable data structures for argument defaults. They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.
```
Comment (kitagry) [inline pyproject.toml:36]: I think it is better not to ignore rules for no particular reason.

**Review by maronuu — APPROVED** [review]
Thank you for fixing the lint settings! LTGM

File: pyproject.toml (hunk 1 above)
Comment (maronuu) [reply pyproject.toml:36]: Commented rules are mistakenly ignored. Thanks for guiding me.

**Review by hirosassa — APPROVED** [review]
Looks good!

**Comment by kitagry** [comment]
Thank you for the review!

**Pull request merged**